app/src/marketplace/components/PriceField/priceField.pcss

```diff
@@ -30,18 +31,19 @@
   outline: none;
   background-color: white;
   border: 0;
-  width: 100%;
+  width: 4rem; /* Mixed with `flex-grow` brings magic to Firefox! ✨ */
```

Comment:
This could be a `0` and `flex-grow` would still take over. Didn't wanna make it 0 though. I guess a non-zero minimum is in good taste. ;) FF doesn't allow input to shrink below the default width thus the change.